src/data/int.rs

```diff
@@ -1,12 +1,6 @@
-use apint::ApInt;
-use traits::Width;
-use digit::Bit;
-use bitwidth::BitWidth;
-use errors::Result;
-use apint::{ShiftAmount};
-use bitpos::{BitPos};
-use uint::UInt;
-use utils::{try_forward_bin_mut_impl, forward_mut_impl, forward_bin_mut_impl};
+use crate::data::{ApInt, UInt};
+use crate::info::{BitWidth, Result, ShiftAmount, BitPos, Width};
+use crate::logic::{try_forward_bin_mut_impl, forward_mut_impl, forward_bin_mut_impl};
```

Comment:
Please restructure this as well with the above mentioned formatting.
This will be part of a future `rustfmt.toml` file and also in `CONTRIBUTIONS.md`.